packs/github/config.schema.yaml

```diff
@@ -0,0 +1,52 @@
+---
+token:
+  description: "GitHub oAuth Token"
+  type: "string"
+  default: "{{user.github_enterprise_oauth_token}}"
+  required: true
+
+user: 
+  description: "GitHub Username"
+  type: "string"
+
+password:
+  description: "GitHub Password"
+  type: "string"
+
+default:
```

Comment:
Cool catch, I'll change it to `github_type` and only have _default_ in the description.